kano_feedback/WidgetWindow.py

```diff
@@ -139,14 +139,16 @@ def _initialise_window(self):
                                           hexpand=False)
         prompt.get_style_context().add_class('prompt')
         prompt.set_justify(Gtk.Justification.LEFT)
-        prompt.set_alignment(0.1, 0.5)
         prompt.set_size_request(410, -1)
         prompt.set_line_wrap(True)
 
-        prompt_container = Gtk.Table(1, 1, False)
-        prompt_container.attach(prompt, 0, 1, 0, 1, Gtk.AttachOptions.SHRINK | Gtk.AttachOptions.FILL)
+        prompt_align = Gtk.Alignment(xalign=0.5, yalign=0.5)
```

Comment:
We still use Gtk.Alignment in loads of other places and it works fine. Until we decide to address the issue of deprecated code entirely, I think it should be fine. It worked for me.